nepomuk/src/main/webapp/WEB-INF/tiles/skeleton.jsp

```diff
@@ -64,6 +64,11 @@
             </ul>
           </li>
           </li>
+               
+          <li role="presentation">
+                <a href="<c:url value="/admin/apps" />" >Apps</a>
+          </li>
```

Comment:
Please remove that for now (or comment out). We'll add that once we're ready to use it.